runhouse/resources/function.py

```diff
@@ -87,11 +87,18 @@ def to(
             >>> rh.function(fn=local_fn).to(gpu_cluster)
             >>> rh.function(fn=local_fn).to(system=gpu_cluster, env=my_conda_env)
         """
+
         if setup_cmds:
             warnings.warn(
                 "``setup_cmds`` argument has been deprecated. "
                 "Please pass in setup commands to the ``Env`` class corresponding to the function instead."
             )
+        if system == "AWS_LAMBDA":
```

Comment:
this logic should prob be moved below where we build the Env object, since that will also be passed in to the constructor

Reply:
makes sense, moving it after creating the env.